.buildkite/hooks/post-checkout

```diff
@@ -0,0 +1,20 @@
+# important: assumes that each agent runs a single job at a time
+
+# ignore errors
+set +e
+
+if [[ $BUILDKITE_AGENT_META_DATA_QUEUE == "pipeline-uploader" ]]; then
+  echo "Skipping local pre-exit on `pipeline-uploader` agents"
+  exit 0
+fi
+
+echo "Cleaning up unused docker containers"
+docker ps -a
+for id in $(docker ps -qa); do
```

Comment:
Why `-a` here? We're only interested in running containers, right?

```
  -a, --all             Show all containers (default shows just running)
```

FWIW, I find it useful to use the long version of arguments in scripts.

Reply:
the goal is to also freeup space. if not important then i can just remove it